pyemma/msm/estimators/_OOM_MSM.py

```diff
@@ -0,0 +1,261 @@
+import numpy as np
+import scipy.linalg as scl
+import scipy.sparse
+from pyemma.util.linalg import _sort_by_norm
+import msmtools.estimation as me
+
+__all__ = ['bootstrapping_count_matrix', 'twostep_count_matrix', 'rank_decision',
+           'oom_components', 'equilibrium_transition_matrix']
+
+
+def bootstrapping_count_matrix(Ct, nbs=500):
+    """
+    Perform bootstrapping on trajectories to estimate uncertainties for singular values of count matrices.
+
+    Parameters
+    ----------
+    Ct : csr-matrix
+        count matrix of the data.
+
+    nbs : int, optional, default=1000
```

Comment:
docs default is not the real default. Just don't document default values, the actual default is visible in the documentation anyway.
Perhaps use the same default here that you have decided to use in the Estimator object (10000)